lib/simulator-xcode-9.js

```diff
@@ -160,6 +149,34 @@ class SimulatorXcode9 extends SimulatorXcode8 {
     }
   }
 
+  /***
+   * Boots simulator and opens simulators UI Client if not already opened.
+   *
+   * @param uiClientPid - process id of simulator UI client.
+   * @param opts - arguments to start simulator UI client with.
+   * @returns {Promise} - a promise that resolves when sim is booted and sim UI client is launched.
+   */
+  async launchSimWindow (uiClientPid, opts) {
```

Comment:
these method don't need the word `Sim` or `Simulator` in them, since the class already has it

Reply:
Ok, will remove the `Sim`s and `Simulators`s .